release-schema.json

```
@@ -75,6 +75,22 @@
           ],
           "minLength": 1
         },
+        "relatedLots": {
+          "title": " Related lot(s)",
+          "description": "The identifiers of related lots that this criterion applies to.",
```

Avoid preposition-stranding, unless it were to make the phrase harder to read (e.g. "a grammatical style up with which I will not put").

```suggestion
          "description": "The identifiers of the lots to which this criterion applies.",
```